commerce_coordinator/apps/frontend_app_payment/serializers.py

```diff
@@ -41,3 +41,17 @@ class GetActiveOrderInputSerializer(serializers.Serializer):  # pylint: disable=
     Serializer for GetActiveOrderView input validation
     """
     edx_lms_user_id = serializers.IntegerField(allow_null=False)
+
+
+class PaymentProcessInputSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+
+    """
+    Serializer for PaymentProcessView input validation
+    """
+    payment_number = serializers.CharField(allow_null=False)
+    order_uuid = serializers.UUIDField(allow_null=False)
+    payment_intent_id = serializers.CharField(allow_null=False)
+    edx_lms_user_id = serializers.IntegerField(allow_null=False)
+    skus = serializers.ListField(
```

Comment:
in an other API, We named it a `product_sku`

Reply:
I think `skus` is the name currently used between ecommerce & frontend-app-payment [[source](https://github.com/openedx/frontend-app-payment/blob/0dd727863ff5362cad052c2c6e4b332ea2518115/src/payment/payment-methods/stripe/service.js#L69)].